default.nix

```diff
@@ -86,12 +86,25 @@ stdenv.mkDerivation rec {
 
   prefixKey = "-prefix ";
 
-  enableParallelBuilding = true;
+  enableParallelBuilding = false;
```

Comment:
Is that still necessary?

Reply:
The guard against parallel jobs is now in the makefile, so I guess Nix does not need to care anymore.

Reply:
Then, it's better to put this back to `true` just in case Nix puts additional guards that would prevent using multiple cores (when Dune could actually take advantage of them).

Reply:
I think this was intended to stop `make -j4` which ends up calling 4 instances of dune, each with 4 jobs. That also doesn't work since dune will tread on its own feet at the moment. I think Emilio added a guard in the makefile, so even if Nix tries to pass -j4, it will not work.

Reply:
I know what it was intended for since I was the one who recommended it.